Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -811,6 +811,16 @@
         "Icon": "/Assets/Helpers.png",
         "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/behaviors/FocusBehaviors.md"
       },
+      {
+        "Name": "AutoSelectBehavior",
+        "Type": "AutoSelectBehaviorPage",
```

Comment:
Basically if you remove this type line here and just use the full path to the `XamlCodeFile` then you don't need the backing page, just be sure to update the XamlOnlyPage with the shadow copy. Thanks!

Reply:
Done !